src/models/lemp11/README.md

```diff
@@ -0,0 +1,57 @@
+# Lemur Pro (lemp11)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Lemur Pro](./img/lemp11.webp)
+
+The System76 Lemur Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 12th Generation (Alder Lake) CPUs
+        - [Core i7-1260P](https://ark.intel.com/content/www/us/en/ark/products/226254/intel-core-i71260p-processor-18m-cache-up-to-4-70-ghz.html)
+        - [Core i5-1240P](https://ark.intel.com/content/www/us/en/ark/products/132221/intel-core-i51240p-processor-12m-cache-up-to-4-40-ghz.html)
+- BIOS
+    - MX25L25673G flash chip running [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E running [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU: Intel Iris Xe Graphics
+        - GOP driver is recommended, VBT is provided
+    - eDP display: 14.0" 1920x1080@60Hz LCD
+        - LCD panel: Innolux N140HCE-EN2 (or equivalent)
+    - External video outputs:
+        - 1x HDMI 2.1
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - Channel 0: 8-GB on-board DDR4
+        - 4x [Samsung K4AAG165WA-BCWE](https://www.samsung.com/semiconductor/dram/ddr4/K4AAG165WA-BCWE/) (2GB each)
```

Comment:
Schematics are wrong. This part is actually P4AAF165WA-BCWDE, per the SPD data provided.

Reply:
@crawfxrd Thanks for the correction! I can't seem to find the new part on [Samsung's website](https://semiconductor.samsung.com/dram/) (their part numbers seem to usually start with `K`), Google's returning zero results, and Bing's not returning any relevant results. Do you know if these (P4AAF165WA-BCWDE) are still Samsung brand and if there's a webpage for them anywhere?

Reply:
Proprietary firmware says it's still Samsung, and the part number is pretty close (K -> P, G -> F, +D).

Here is the SMBIOS data:

```
Handle 0x0021, DMI type 17, 92 bytes
Memory Device
	Array Handle: 0x0020
	Error Information Handle: Not Provided
	Total Width: 64 bits
	Data Width: 64 bits
	Size: 8 GB
	Form Factor: SODIMM
	Set: None
	Locator: Controller0-ChannelA
	Bank Locator: BANK 0
	Type: DDR4
	Type Detail: Synchronous
	Speed: 3200 MT/s
	Manufacturer: Samsung
	Serial Number: 00000000
	Asset Tag: 9876543210
	Part Number: P4AAF165WA-BCWDE    
	Rank: 1
	Configured Memory Speed: 3200 MT/s
	Minimum Voltage: Unknown
	Maximum Voltage: Unknown
	Configured Voltage: 1.2 V
	Memory Technology: DRAM
	Memory Operating Mode Capability: Volatile memory
	Firmware Version: Not Specified
	Module Manufacturer ID: Bank 1, Hex 0xCE
	Module Product ID: Unknown
	Memory Subsystem Controller Manufacturer ID: Unknown
	Memory Subsystem Controller Product ID: Unknown
	Non-Volatile Size: None
	Volatile Size: 8 GB
	Cache Size: None
	Logical Size: None
```

In fact, it looks like the lemp10 also is a different part according to the SPD data (M471A1G44AB0-CWE).

Reply:
> In fact, it looks like the lemp10 also is a different part according to the SPD data (M471A1G44AB0-CWE).

I physically confirmed the lemp10's part number with a photo here: https://github.com/system76/tech-docs/pull/49#issuecomment-747610268 

The photo shows lemp10 had K4AAG165WA-BCWE. Are you sure this isn't a matter of a more internal part number vs. an overarching model number that Samsung would advertise?

That PR did say the printed part number (which is what we went with) was different from what was in the Clevo manual, so maybe getting a photo of the lemp11's RAM chips on the back of the motherboard would be a good idea.)

Reply:
Can't see shit in the service manual, and I'm not taking it out, so I don't know what is actually attached to the board.

Considering I can make it say anything in the SPD data, it might be an internal/config thing.

Reply:
@leviport took apart our lemp11 and got a picture:

![PXL_20220701_192934339](https://user-images.githubusercontent.com/7199422/176959229-848e396c-f775-4613-af94-677bfbdb68cb.jpg)

Looks like K4AAG165WA-BCWE is correct for lemp11, same as lemp10.